hudi-common/src/main/java/org/apache/hudi/io/storage/HoodieParquetConfig.java

```diff
@@ -78,4 +83,12 @@ public T getWriteSupport() {
   public boolean dictionaryEnabled() {
     return dictionaryEnabled;
   }
+
+  public long getMinRowCountForSizeCheck() {
+    return minRowCountForSizeCheck;
```

Comment:
How the user config these options, seems we only use the default value, if that is the case, I would suggest we hard code the options in `HoodieBaseParquetWriter`